astrodb_utils/sources.py

```diff
@@ -43,8 +44,8 @@ def find_source_in_db(
         Right ascensions of sources. Decimal degrees.
     dec: float
         Declinations of sources. Decimal degrees.
-    search_radius
-        radius in arcseconds to use for source matching
+    search_radius: u.Quantity
+        Search radius. Default is 60 arcseconds.
```

Comment:
Do you want an `if isinstance` check of some kind to convert to a Quantity if the user provides a float? Downstream packages (eg, SIMPLE) might be supplying floats.

Reply:
I'm fine ignoring this for now and letting downstream packages address it (I think astrodbkit promotes to Quantity if it isn't already)